shanoir-ng-users/src/main/java/org/shanoir/ng/email/EmailServiceImpl.java

```diff
@@ -537,7 +537,7 @@ public void notifyStudyManagerAccessRequest(AccessRequest createdRequest) throws
 				};
 				// Send the message
 				LOG.info("Sending study-users-added mail to {} for study {}", studyAdmin.getUsername(), createdRequest.getStudyId());
-				mailSender.send(messagePreparator);
+				// mailSender.send(messagePreparator);
 			}
         } else {
```

Comment:
I would rather place the error condition at the beginning of the method in a simple if : 

``` java
if (CollectionUtils.isEmpty(studyAdmins)) {
   LOG.error("No admins for this study. This should not happen.");
   throw new ShanoirException("No admin could be found fot this study.");
}

for (User studyAdmin : studyAdmins) {
...
```